pom.xml

```diff
@@ -1850,15 +1852,66 @@
                     <artifactId>maven-bundle-plugin</artifactId>
                     <version>${osgi.bundle.plugin.version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-clean-plugin</artifactId>
+                    <configuration>
+                        <followSymLinks>false</followSymLinks>
+                        <!-- Skip this step if on windows -->
+                        <skip>${js.skip.clean}</skip>
+                        <filesets>
+                            <fileset>
+                                <directory>${basedir}</directory>
```

Comment:
I've copied this code, but I was kind of expecting this should be `project.basedir`. Or are both defined?